spec/models/event_spec.rb

```diff
@@ -23,5 +36,11 @@
       event = create(:event, author: author)
       expect(event.author).to eq author
     end
+
+    it "does not validate when 'body' is blank" do
+      event = Event.new(title: nil)
```

Comment:
I might have missed the point here, but should that be `body: nil` not `title: nil`?